### DISCUSSION TITLE
Desgin page keep loading, still exists in Budibase version 2.4.4 (Docker deployment).

### DISCUSSION THREAD

**Discussion by yarin-zhang** [Budibase/budibase#9978] [Help]
This is a follow-up to an unresolved Issue: #7749.

This problem still exists in Budibase version 2.4.4 (Docker deployment).

I found a very strange phenomenon, when I am in Firefox Developer Edtion, the page is displayed normally.

![image](https://user-images.githubusercontent.com/58888890/224204267-6071c7e8-53ed-42e5-91cb-f1f33c987c1d.png)

But when I'm in Chrome it keeps loading. I tried guest mode (without any browser plugins) and it also keeps loading.

![image](https://user-images.githubusercontent.com/58888890/224204464-71fa306c-b5b7-463b-8189-2ebeed7b4a46.png)

_Originally posted by @yarin-zhang in https://github.com/Budibase/budibase/issues/7749#issuecomment-1463103699_

**Comment by aptkingston** [comment]
Hi @yarin-zhang. The issue you're encountering is due to you running Budibase from source, and the solution is described in our dev setup documentation: https://github.com/Budibase/budibase/blob/develop/docs/DEV-SETUP-DEBIAN.md#file-descriptor-issues-with-vite-and-chrome-in-linux

**Reply by frnksmdlkedjnfr** [reply]
Tried adding that line into /etc/security/limits.conf, in both WSL 2 and the Debian inside the Budibase container. Still doesn't work. Design page loads infinitely on Chrome but not on Firefox.

**Comment by aptkingston** [comment]
I'm also converting this to a discussion because it's concerning dev setup, and the solution is listed in our docs.

**Comment by yarin-zhang** [comment]
Thanks, I will try the solution mentioned there, looks like a Chrome issue? 

Is it possible to get this solved from Docker sources?

**Comment by slinkytoad** [comment]
Found this post after updating to 2.5.0. I run budibase behind NGINX and had `add_header Referrer-Policy 'origin'` in my config. I commented that out, and it's working again.

**Reply by frnksmdlkedjnfr** [reply]
How exactly did you do that?

**Reply by slinkytoad** [reply]
I am running NGINX as my reverse proxy outside of the budibase app. In that config I had the above mentioned line, as it is common for any reverse proxy I setup. 

The config file for NGINX is usually located at `/etc/nginx/sites-available/` but yours may vary.

**Reply by frnksmdlkedjnfr** [reply]
I was not able to find a config file that contains the 'add_header' module u mentioned

**Reply by slinkytoad** [reply]
When you go to access Budibase, are you going to port 80/443 or 10000?

**Reply by slinkytoad** [reply]
I suppose another question, how did you deploy budibase?

**Reply by frnksmdlkedjnfr** [reply]
I deployed it with Docker, as per BB's official docs. Normal docker run stuff:
`docker run -d -t \
  --name=budibase \
  -p 10000:80 \
  -v /local/path/data:/data \
  --restart unless-stopped \
  budibase/budibase:latest`

This is it. I am running docker inside WSL 2 Debian. After the deployment, I was able to open the main page. Everything except rendering the actual app worked fine. Even a ready-made template won't open. It just keeps loading. But as everyone else experiences, everything runs perfectly fine on Firefox.

I've already tried the official fix by adding a file limit, just to no avail.

**Reply by slinkytoad** [reply]
I am not sure then on this. I didn't run mine via WSL.

**Reply by frnksmdlkedjnfr** [reply]
OK I just tested this on a physical machine, running Debian. The problem persists, Chrome just won't open the design page.
